docs/source/developing/releasing.adoc

```diff
@@ -29,3 +37,19 @@ defined in your environment as _GITHUB_ACCESS_TOKEN_.
 ----
 $ make release
 ----
+
+== Post release
+
+As part of the release process we also send an release announcement and edit the GitHub
+release page.
+
+For the latter we usually add a categorized list of resolved issues as well as some
+release highlights (most often taken from the release announcement).
+
+If you have the link:https://stedolan.github.io/jq/[jq] installed on your machine, you
+can use the link:https://github.com/minishift/minishift/blob/master/scripts/release/issue-list.sh[issue-list.sh] script to generate the markdown needed for the adding the
+issue list, for example:
+
+----
+$ issue-list.sh -r minishift -m 9
```

Comment:
@hferentschik , but if you mention the command like that and don't tell to cd to `scripts/release` folder, user will try from minishift repo folder. Thats what I have found :)